### DISCUSSION TITLE
Query using the plan_batch api

### DISCUSSION THREAD

**Discussion by shannonwarren** [NVlabs/curobo#273] [Research Q&A]
The current plan_batch api takes in a start JointState and goal_set which is a Pose. Is there a plan to support a goal set of JointStates? I was curious about this functionality since my task sequencing pipeline takes a set of goal JointStates

**Comment by peter-mitrano-bg** [comment]
I think you want `plan_batch_goalset` maybe?

**Reply by shannonwarren** [reply]
Plan batch goalset needs goal poses too. I need an api that takes a set of goal angles.
